app/models/rubygem.rb

```diff
@@ -23,6 +23,8 @@ class Rubygem < ActiveRecord::Base
 
   # TODO: Remove this once we move to GemDownload only
   after_create :create_gem_download
+  after_create :set_tsvector
```

Comment:
Let me know if it okay to use trigger here. [Blog](https://robots.thoughtbot.com/optimizing-full-text-search-with-postgres-tsvector-columns-and-triggers#cAAcAA) I followed used trigger and [pg_search suggests](https://github.com/Casecommons/pg_search/blob/e4e2ed236690e27337a8ed0d31d38d7721f35a5c/README.md#using-tsvector-columns) use of trigger when using tsvector.

I don't feel good about triggers mostly because the code for it exists in the migration files.